tests/Endpoints/AbstractEndpoint.php

```diff
@@ -51,10 +56,14 @@ public function setUp(): void
 
         $authFixtures = [
             LoadAuthenticationData::class,
+            LoadSchoolData::class,
+            LoadServiceTokenData::class,
         ];
         $testFixtures = $this->getFixtures();
         $fixtures = array_merge($authFixtures, $testFixtures);
-        $this->databaseTool = self::getContainer()->get(DatabaseToolCollection::class)->get();
+        /* @var DatabaseToolCollection $databaseToolCollection */
```

Comment:
i tried to appease the API here.

![image](https://github.com/ilios/ilios/assets/1410427/6416f23f-c0e6-4bfc-bc29-66a23f1fbbb2)

but you're right - PHPStan is not complaining so it can be removed.

yanked.